src/sql/parts/dashboard/widgets/insights/views/charts/chartInsight.component.ts

```diff
@@ -348,3 +356,13 @@ function isValidData(data: IInsightData): boolean {
 
 	return true;
 }
+
+Chart.pluginService.register({
+	beforeDraw: function (chart) {
```

Comment:
Does setting the options guarantee a redraw of the background?

Reply:
This is a standard event from chartjs for updating charts before the drawing event.
It should be noted that this is a global event that will impact all chartjs charts in the solution:
https://www.chartjs.org/docs/latest/developers/plugins.html#global-plugins

I have tested to change the theme of sqlops while showing a chart and the background is updated as expected.